### ISSUE TITLE
feedstock: Admin Web Service command to add new Python 3.11 build?

### ISSUE THREAD

**Issue by megies** [conda-forge/conda-forge.github.io#1846]
### Your question:

I was wondering how to update a feedstock, such that a new Python version (here: 3.11) is added to the build matrix. I could not find anything in the conda-forge docs, specifically I was looking in the [admin web services commands list](https://conda-forge.org/docs/maintainer/infrastructure.html#admin-web-services), because aside from updating the recipe this is the only thing I ever need to maintain our feedstock (actually just ever need the "please rerender" command).

I can see form looking at past new Python version additions that at some point @regro-cf-autotick-bot [seems to open a PR automatically](https://github.com/conda-forge/obspy-feedstock/pull/48), adding new Python versions, but I can not find if/when/how that gets triggered.

Also I am wondering if this won't get triggered because our latest stable package metadata on pypi does not include 3.11 in the metadata, simply because 3.11 wasn't out when we did the last release.

### TL;DR:
Is there any place with a "How To add New Python version" for feedstocks? Can't find one and pretty clueless what to do to get our current release packaged for new Python 3.11

**Comment by chrisburr** [comment]
This is done automatically using the infrastructure for "migrations". This causes the bot to issue PRs to feedstocks in dependency order and the status can be tracked on the conda-forge status page:
 
https://conda-forge.org/status/#python311

I guess you're asking about `obspy` in which case it's currently in the list of feedstocks that are waiting for a PR so it should come in the next few hours. (PR creation is rate limited to avoid overloading the CI and making conda-forge unusable.)

**Comment by megies** [comment]
Thanks a lot for the quick response and clarification @chrisburr!

I'm proposing to add info on this in the maintainer docs, feel free to change as pleased or close without merging
